docker/maistra-builder_2.6.Dockerfile

```diff
@@ -55,23 +56,15 @@ RUN dnf -y upgrade --refresh && dnf -y install --setopt=install_weak_deps=False
     python3-devel \
     python3-pip python3-setuptools \
     wget jq rsync \
-    compat-openssl11-1:1.1.1k-4.el9_0 \
+    compat-openssl11 \
     libstdc++-static \
     libxcrypt-compat-0:4.4.18-3.el9 \
     iptables-nft libcurl-devel \
     git less rpm rpm-build gettext file \
     iproute ipset rsync net-tools \
     ninja-build \
-    sudo autoconf automake cmake unzip wget xz procps
-
-# Install libbpf-devel from centos stream
-# Do not run "dnf -y upgrade --refresh" again. Avoid conflicts between ubi and centos dnf upgrade.
-RUN set -eux; \
-    dnf -y install --setopt=install_weak_deps=False \
-    "https://mirror.stream.centos.org/9-stream/BaseOS/$(uname -m)/os/Packages/libzstd-1.5.0-2.el9.$(uname -m).rpm" \
-    "https://mirror.stream.centos.org/9-stream/AppStream/$(uname -m)/os/Packages/libzstd-devel-1.5.0-2.el9.$(uname -m).rpm" \
-    "https://mirror.stream.centos.org/9-stream/AppStream/$(uname -m)/os/Packages/elfutils-libelf-devel-0.189-3.el9.$(uname -m).rpm" \
-    "https://mirror.stream.centos.org/9-stream/CRB/$(uname -m)/os/Packages/libbpf-devel-1.2.0-1.el9.$(uname -m).rpm"
+    sudo autoconf automake cmake unzip wget xz procps dnf-plugins-core \
+    libbpf-devel-2:1.1.0-2.el9
```

Comment:
Why are you specifying a version here? Why not just `libbpf-devel`?

Reply:
We should not rely on fedora's packages to build or test our software. We should use the builder container for that, and we should make sure the builder container has all the necessary packages with the correct versions to build&test our software.